rest.py

```diff
@@ -92,7 +88,7 @@ def __connect(self):
 
     def __generate_cookie(self):
         # use env to have a clean environment
-        command = 'rm -f %s; env -i KRB5CCNAME="$KRB5CCNAME" cern-get-sso-cookie -u %s -o %s --reprocess --krb' % (self.cookie, self.server, self.cookie)
+        command = 'rm -f %s; env -i KRB5CCNAME="$KRB5CCNAME" auth-get-sso-cookie -u %s -o %s' % (self.cookie, self.server, self.cookie)
```

Comment:
I'm a bit puzzled by this tbh. Why do we need to explicitly tell the script to use the system default?

Reply:
I think it is because some runtime environments seem to use other CA bundles based on the content given by "certifi" package (the last is a theory I am not completely sure. I am referencing a comment that explains it better: [link](https://github.com/CMSCompOps/WmAgentScripts/pull/1203#issuecomment-1672754502))